packages/ckb-indexer/src/collector.ts

```diff
@@ -234,10 +245,64 @@ export class IndexerCollector implements BaseCellCollector {
     return counter;
   }
 
+  private async request(ckbIndexerUrl: string, data: unknown): Promise<any> {
+    const res: rpcResponse = await axios.post(ckbIndexerUrl, data);
+    if (res.status !== 200) {
+      throw new Error(`indexer request failed with HTTP code ${res.status}`);
+    }
+    if (res.data.error !== undefined) {
+      throw new Error(
+        `indexer request rpc failed with error: ${JSON.stringify(
+          res.data.error
+        )}`
+      );
+    }
+    return res.data;
+  }
+
+  private async getLiveCellWithBlockHash(lastCursor?: string) {
+    if (!this.otherQueryOptions) {
+      throw new Error("CKB Rpc URL must provide");
+    }
+    let result: GetCellsResults = await this.getLiveCell(lastCursor);
+    if (result.objects.length === 0) {
+      return result;
+    }
+    const requestData = result.objects.map((cell, index) => {
+      return {
+        id: index,
+        jsonrpc: "2.0",
+        method: "get_block_hash",
+        params: [cell.block_number],
+      };
+    });
+    const blockHashList: any[] = await this.request(
```

Comment:
```suggestion
    // TODO define type here
    const blockHashList: any[] = await this.request(
```

Reply:
feat(ckb-indexer): add type to blockHash list